math/src/field/f64/mod.rs

```diff
@@ -52,7 +52,7 @@ const ELEMENT_BYTES: usize = core::mem::size_of::<u64>();
 ///
 /// Internal values represent x * R mod M where R = 2^64 mod M and x in [0, M).
 /// The backing type is `u64` but the internal values are always in the range [0, M).
-#[derive(Copy, Clone, Debug, Default)]
+#[derive(Copy, Clone, Debug, Default, Eq, PartialOrd, Ord)]
```

Comment:
I think deriving `PartialOrd` directly relies on lexicographic ordering, which has little meaning when dealing with possibly unreduced elements or in Montgomery form?

Reply:
Do you have any recommendations on how to implement this properly, if at all possible?

Reply:
I would assume comparing reduced elements is the way to go (same as in implementations of `PartialEq`) though maybe @irakliyk has a different solution?